docs/extensions/overview.md

```diff
@@ -26,8 +14,41 @@ INSTALL 'fts';
 LOAD 'fts';
 ```
 
-<!--
-TODO: How to check which extensions have been loaded or are available
+If you are using the Python API client, you can install and load them with the `load_extension(name: str)` and `install_extension(name: str)` methods.
 
-https://github.com/duckdb/duckdb/search?q=loaded_extensions
+## Listing core extensions
```

Comment:
Feedback on these headers is very welcome 😅

Reply:
Perhaps just `Listing extensions`? It lists all installed extensions regardless of their source.